packages/libs/error-stack/tests/snapshots/test_debug__full__complex.snap

```diff
@@ -0,0 +1,52 @@
+---
+source: tests/test_debug.rs
+expression: "redact(&format!(\"{report:?}\"))"
+---
+Context A
+├╴tests/test_debug.rs:494:14
+├╴Printable A
+│
+╰┬▶ Root error
+ │  ├╴tests/common.rs:147:5
+ │  ├╴Printable A
+ │  ├╴backtrace with [n] frames (1)
+ │  ╰╴spantrace with 2 frames (1)
+ │
+ ├▶ Root error
+ │  ├╴tests/common.rs:147:5
+ │  ├╴Printable B
+ │  ├╴Test
+ │  ├╴backtrace with [n] frames (2)
+ │  ├╴spantrace with 2 frames (2)
+ │  ╰╴1 additional opaque attachment
+ │
+ ╰▶ Root error
+    ├╴tests/common.rs:147:5
+    ├╴Printable B
+    ├╴Test
+    ├╴backtrace with [n] frames (3)
+    ├╴spantrace with 2 frames (3)
+    ╰╴3 additional opaque attachments
+
+━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
+
+Backtrace No. 1
+   [redacted]
+
+
+Span Trace No. 1
+   [redacted]
+
+Backtrace No. 2
+   [redacted]
+
+
+Span Trace No. 2
+   [redacted]
+
+Backtrace No. 3
+   [redacted]
+
+
+Span Trace No. 3
+   [redacted]
```

Comment:
There are different amount of newlines between the snippets. Probably this is the same reason as in https://github.com/hashintel/hash/pull/794/files#r946976913